tests/__snapshots__/files/prism-core.js.shot

```diff
@@ -587,8 +589,8 @@ var Prism = (function (_self) {
 		},
 
 		/**
-		 * Low-level function, only use if you know what you’re doing. It accepts a string of text as input and the
-		 * language definitions to use, and returns a string with the HTML produced.
+		 * Low-level function, only use if you know what you’re doing. It accepts a string of text as input and the language
```

Comment:
I don't know why this changed. It seems like something got fixed along the way but I can't tell you what. However, the change is correct. The previous text was wrapped too early.